test/types/TestAlternativesPerformance.cpp

```diff
@@ -17,7 +17,7 @@ TEST(TestAlternativesPerformance, TestConstructorBaseConstructor) {
       os.str(),
       "AlternativesPerformance( PerformanceTable[ Performance: Perf( name : "
       "a0, crit : crit0, value : 0 ) Perf( name : a0, crit : crit1, value : 0 "
-      ") | ], AlternativesAssignment{ a0->Category(id : , rank : -1) }");
+      ") | ], AlternativesAssignment{ {, (a0: Category(id : , rank : -1)),}");
```

Comment:
I think this is related to the << operator in utils, but it is not easy to read here, and why is there random **,** at the begining and the end of the AA? 
Again, I think there are too many { for not enough } here